core/src/main/java/com/linecorp/armeria/client/endpoint/healthcheck/HttpHealthCheckedEndpointGroupBuilder.java

```diff
@@ -52,6 +54,17 @@ public HttpHealthCheckedEndpointGroupBuilder(EndpointGroup delegate, String heal
         this.healthCheckPath = requireNonNull(healthCheckPath, "healthCheckPath");
     }
 
+    /**
+     * If used, a health check request will be sent to the specified port instead of the port of the
```

Comment:
Sorry but how about rephrasing a little bit:

```
Sets the port where a health check request will be sent instead of the original port number
specified by {@link EndpointGroup}'s {@link Endpoint}s. This property is useful when your
server listens to health check requests on a different port.
```